docroot/sites/its.uiowa.edu/modules/its_core/css/related-service.css

```diff
@@ -0,0 +1,3 @@
+.article--meta.block-field-blocknodearticlefield-article-related-service {
+  display: block;
```

Comment:
Would it make sense to have a utility class like `display-block` in uids_base that would remove the need for this separate library/css?

Reply:
I removed this because it is no longer needed after layout placement adjustment.

Reply:
This may not be necessary after Ben's changes, so lets see what happens there first then move from there